src/app/GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -163,6 +163,7 @@ public void ExpandToFile(string filePath, int? line, bool requestBlame)
         public void InitSplitterManager(SplitterManager splitterManager)
         {
             splitterManager.AddSplitter(FileTreeSplitContainer, "FileTreeSplitContainer");
+            BlameControl.InitSplitterManager(splitterManager, "RevisionFileTree");
```

Comment:
There is no need to adapt this because the `RevisionFileTreeControl` class will be deleted when it will be replaced with `RevisionDiffControl` in the near future.

Reply:
Doesn't hurt though :)